examples/csharp/integrationTest/Rest/ApiIntegrationTest.cs

```diff
@@ -5,14 +5,13 @@
 using Twilio.Rest.Api.V2010;
 using Twilio.Rest.Api.V2010.Account;
 using Twilio.Rest.Api.V2010.Account.Call;
-using Twilio.Rest.Api.V2010.Credential;
 
 namespace Twilio.Tests.Rest
 {
     [TestFixture]
     public class ApiIntegrationTest
     {
-        readonly string baseURL = "http://prism_twilio:4010";
+        readonly string baseURL = "http://prism_api_v2010:80";
```

Comment:
This should be `http://api.twilio.com`

Reply:
Changed baseURL to http://api.twilio.com